### DISCUSSION TITLE
How to measure thrust in PWM

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1337] [Q&A]
Hello. I'm conducting an experiment to measure the relationship between Thrust and PWM as detailed in the provided link.

https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/pwm-to-thrust/

My current method aligns with the document. However, despite supplying sufficient PWM to hover, the weight measured by the scale doesn't decrease significantly.

I've mounted the Crazyflie on a 3D printed jig using the holes on its deck and placed it on a scale to measure weight loss.

Here's a video of my experiment.

https://github.com/bitcraze/discussions/assets/143582301/e4094588-5953-446f-a0c9-e710fad2cb14

![KakaoTalk_20240513_105211962_01](https://github.com/bitcraze/discussions/assets/143582301/b53e2f28-95e1-43eb-b8e1-7834ac78d918)

Could you please advise if there are any issues with my experimental approach and how to address them?

**Comment by tobbeanton** [comment]
The problem with your setup is that you don't have a free airflow which is going to effect the measurements. A better way to do it is to measure in a push fashion instead by swapping the CW and CCW propellers.

**Reply by WoodyloveBo** [reply]
As I understand it，Are you suggesting measuring it by loading the payload onto the top of the Crazyflie?

**Reply by tobbeanton** [reply]
A pushing fashion is so that the crazyflie pushes down on the scale instead of lifting it. For that you need to swap the propellers.
